apps/dashboard/app/javascript/path_selector/path_selector_data_table.js

```diff
@@ -92,15 +92,22 @@ export class PathSelectorTable {
       this._table.rows.add(data.files);
       this.setLastVisited(data.path);
       this._table.draw();
-      $('#loading-icon').hide();
-      $(`#${this.tableId}_wrapper`).show();
+      this.resetTable();
     } catch (err) {
-      $('#loading-icon').hide();
-      $(`#${this.tableId}_wrapper`).show();
+      this.resetTable();
+      if (err.message.match("Permission denied")) {
+        $('#forbidden-warning').show();
```

Comment:
I think we should also focus this element at this point. If you've scrolled down past it, you can't really see it. I had to set the `tabindex=0` to get it to work. 

Also note that `focus()` is deprecated so we'd want `.trigger("focus")` (in jQuery).